transformations/color_transformation/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "color_transformation",
+  "test_cases": [
+    {
+      "class": "ColorTransformation",
+      "inputs": {
+        "sentence": "I am feeling blue today."
+      },
+      "outputs": [{
+        "sentence": "I am feeling misty rose today."
+      }]
+    },
+    {
+      "class": "ColorTransformation",
+      "inputs": {
+        "sentence": "I bought this pink shoes today! Isn't it pretty?"
```

Comment:
* shoes --> shoe

Reply:
Or change this to these instead.

Reply:
Thanks for the catch! Changed it to "pair of shoes."